example.js

```diff
@@ -19,6 +21,13 @@ class CheckFeatures {
   }
 
   process(tweet) {
+    if (tweet.place) {
+      var hashtagList = tweet.entities.hashtags
+      for (var i = 0; i < hashtagList.length; i++) {
+        console.log('hashtag: ' + hashtagList[i]);
+        db.insertHashtagStats(tweet.place.country, hashtagList[i].text);
+      }
+    }
     if (tweet.geo) {
       this.placeCount += 1;
     }
```

Comment:
It looks like we're making this file the starting point for the batch scanning job, instead of just an example. Maybe rename this file `scan.js` or `scanjob.js` or something.